### PR TITLE
oasdiff diff and check-breaking actions

### PR DESCRIPTION
**Pull request by effoeffi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+    format: 'text'
+    fail-on-diff: false
+```
+
+### Check for breaking API changes, and fail if any are found
+Copy and paste the following snippet into your build .yml file:
+```
+- name: Running OpenAPI Spec diff action
+  id: test_ete
+  uses: ./check-breaking
```
Comment (reuvenharrison) [inline README.md:27]: Should be full path?

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+## How to use?
+Depend on your use case:
+
+### Find diff
+Copy and paste the following snippet into your build .yml file:
+```
+- name: Running OpenAPI Spec diff action
+  id: test_ete
+  uses: ./diff
```
Comment (reuvenharrison) [inline README.md:14]: Should be full path?

**Review by reuvenharrison — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (effoeffi) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (effoeffi) [reply README.md]: fixed

**Pull request merged**